### PR TITLE
Terraform config for stress test runner

### PR DESCRIPTION
**Pull request by mberhault**
[Review on Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/4103)

### CODE REVIEW

File: cloud/aws/stress/supervisor.conf [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+[supervisorctl]
+serverurl=http://127.0.0.1:9001 ; use an http:// url to specify an inet socket
+
+[program:stress]
+directory=%(here)s
+command=%(here)s/stress.sh
+process_name=%(program_name)s
+numprocs=1
+autostart=false
+autorestart=false
```
Comment (bdarnell) [inline cloud/aws/stress/supervisor.conf]: What value does `supervisord` add if it's not restarting the process?

**Comment by bdarnell** [comment]
LGTM

File: cloud/aws/stress/supervisor.conf (hunk 1 above)
Comment (mberhault) [inline cloud/aws/stress/supervisor.conf]: I mostly use it to monitor that all processes are still running (this is done in the cockroach-prod script). Very useful, especially when there's more than one node.

**Comment by BramGruneir** [comment]
LGTM

File: cloud/aws/stress/stress.sh [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+set -x
+
+LOG_DIR="logs"
+NUM_PROCS=4
+MAX_RUNS=0
+MAX_TIME=15m
+MAX_FAILS=1
+
+STRESS_FLAGS="-p ${NUM_PROCS} -maxruns ${MAX_RUNS} -maxtime ${MAX_TIME} -maxfails ${MAX_FAILS} -stderr"
```
Comment (BramGruneir) [inline cloud/aws/stress/stress.sh]: Actually, you could remove the `-p ${NUM_PROCS}` and just let stress default to the number of processors.

Comment (mberhault) [inline cloud/aws/stress/stress.sh]: done.

**Pull request merged**